packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -1269,6 +1334,26 @@ export enum InstanceSize {
    * Instance size METAL (metal)
    */
   METAL = 'metal',
+
+  /**
+   * Instance size XLARGE16METAL (metal-16xl)
+   */
+  XLARGE16METAL = 'metal-16xl',
+
+  /**
+   * Instance size XLARGE24METAL (metal-24xl)
+   */
+  XLARGE24METAL = 'metal-24xl',
+
+  /**
+   * Instance size XLARGE32METAL (metal-32xl)
+   */
+  XLARGE32METAL = 'metal-32xl',
+
+  /**
+   * Instance size XLARGE48METAL (metal-48xl)
+   */
+  XLARGE48METAL = 'metal-48xl',
```

Comment:
do these not need to be in the list below?

Reply:
If you mean in the `instanceClassMap`, they only contain instance classes, the `metal-*` properties are instance sizes